gqlalchemy/models.py

```diff
@@ -365,12 +367,20 @@ def field_in_superclass(field, constraint):
         else:
             cls.labels = {cls.label}
 
+        db = kwargs.get("db", None)
+        if cls.index is True:
+            if db is None:
+                raise GQLAlchemyDatabaseMissingInNodeClassError(cls=cls)
+            index = MemgraphIndex(cls.label)
+            db.create_index(index)
+
         for field in cls.__fields__:
             attrs = cls.__fields__[field].field_info.extra
             field_type = cls.__fields__[field].type_.__name__
 
             label = attrs.get("label", cls.label)
-            db = attrs.get("db", None)
+            if db is None:
+                db = attrs.get("db", None)
```

Comment:
And again, here You don't have it

Reply:
I'm missing something, where should the newline be?